### DISCUSSION TITLE
Improved header depth indictor

### DISCUSSION THREAD

**Discussion by tauren** [Akifyss/obsidian-border#74] [Ideas]
I really appreciate the visual cues for determining header depth in this theme. The shrinking font sizes are so close it is hard to tell which is which. The color coding is nice, but after using the theme for a week, I still don't intuitively know the colors yet. But the `H1`, `H2`, etc labels are SO GOOD!

<img width="142" alt="image" src="https://github.com/Akifyss/obsidian-border/assets/282084/5a550f84-de59-4374-a8dd-6dced4101156">

Would it be possible to change the colored bar before the header text to have the multiple lines based on the header depth? That would visually indent the header further as the depth increased. And the number of lines would match the depth as well. Like this:

<img width="76" alt="image" src="https://github.com/Akifyss/obsidian-border/assets/282084/de62e200-2ab2-450c-96ff-5273657d4776">
 
I haven't checked how the colored line is being created, but if you are using CSS borders, doing multiple lines wouldn't be an easy change. However, it would be a nice touch and very helpful.

Additionally, it would be really nice if Edit mode could be improved with visual cues as well. I know I can just count the grey hash marks, but drawing a colored line before the hash marks, colorizing the hash marks, adding the `H1` hover icon, or whatever would help. I don't think everything would be needed, or even possible due to the collapse carrot icon, but a visual cue would be appreciated. 

<img width="183" alt="image" src="https://github.com/Akifyss/obsidian-border/assets/282084/be86a136-4133-4c38-b0b8-4f977dfb173c">

**Comment by Akifyss** [comment]
I will follow up on your suggestion to explore the feasibility, but I am not sure it will be implemented for sure. Currently you can adjust the header (size, color) via the style settings plugin to increase header distinction.

I'm not sure if I misunderstood you, but what I understand now is that in edit mode, your header doesn't have the colored line or the H1 hover icon? Because in the normal case, they are also present in edit mode:

![image](https://github.com/Akifyss/obsidian-border/assets/23567154/089cd6ae-02b7-48c3-8d74-ba15eb05e12f)
![image](https://github.com/Akifyss/obsidian-border/assets/23567154/7cc47a41-aa69-4bc3-bc3e-16f450fb9bfd)
